src/main/java/winstone/ServerConnectorBuilder.java

```diff
@@ -0,0 +1,84 @@
+package winstone;
+
+import org.eclipse.jetty.server.ForwardedRequestCustomizer;
+import org.eclipse.jetty.server.HttpConfiguration;
+import org.eclipse.jetty.server.HttpConnectionFactory;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.ServerConnector;
+import org.eclipse.jetty.util.ssl.SslContextFactory;
+
+public class ServerConnectorBuilder {
+
+    private int listenerPort;
+    private int keepAliveTimeout;
+    private int acceptors;
+    private int selectors;
+    private int requestHeaderSize;
+    private String listenerAddress;
+    private Server server;
+    private SslContextFactory sslContextFactory;
+
+    public ServerConnectorBuilder withListenerPort(int listenerPort) {
+        this.listenerPort = listenerPort;
+        return this;
+    }
+
+    public ServerConnectorBuilder withKeepAliveTimeout(int keepAliveTimeout) {
```

Comment:
AFAICT this is not being called?

Reply:
Thank you for the hint, I forgot to add the parameter there.